dataflow/src/org/checkerframework/dataflow/analysis/BackwardTransferFunction.java

```diff
@@ -0,0 +1,13 @@
+package org.checkerframework.dataflow.analysis;
+
+import java.util.List;
+import org.checkerframework.dataflow.cfg.UnderlyingAST;
+import org.checkerframework.dataflow.cfg.node.ReturnNode;
+
+public interface BackwardTransferFunction<V extends AbstractValue<V>, S extends Store<S>>
```

Comment:
Interfaces should have documentation. See what parts of the existing ForwardTF can me used for TF and BackwardTF. Put the common stuff into TF and then the specifics into the subclass javadoc.